### DISCUSSION TITLE
Bépo alternative

### DISCUSSION THREAD

**Discussion by driccio** [manna-harbour/miryoku#167] [Ideas]
Hello all,

I am planning to use Bépo (a french layout, [see here](https://bepo.fr/wiki/Pr%C3%A9sentation)) on a 3*5 keys by hand keyboard.

Bépo has the specificity to not be a simple reordering of qwerty keys. All is totaly different. Even if QMK did a nice work to manage that layout (I don't know for other frameworks), there is some work to do to be managed with Miryoku, because :
- Symbols layer need a total lifting
- Numbers layer also need to be reviewed (this part is not really a problem).

As we are on a 3*5 keys by hand, we are missing keys to map all Bépo specified on the base layer. So it needs some subjective choices to reduce the base layer. I found some tries from community and have wrote a first layout to try.

So before to go further, I am wondering if that work on Miryoku was already done by others and, if yes, if you have pointers on that work. After search I find nothing specific to Miryoku.

Thanks a lot for ready me.

Damien

**Comment by manna-harbour** [comment]
I don't have a specific solution, but for general approaches see https://github.com/manna-harbour/miryoku/discussions/152#discussioncomment-4073122. Also see the [Mechanical Keyboards France discord](https://discord.gg/E6D3jTHBj2) if you're not there already, as there seem to be a few Miryoku users there.

**Comment by driccio** [comment]
Hi,

I pushed my work about that on [my repo](https://github.com/driccio/miryoku_qmk/tree/bepo/users/manna-harbour_miryoku). 

I tried to be the less intrusive about my changes in your files. As suggested [here](https://github.com/manna-harbour/miryoku/discussions/171) it would be nice to customize our configuration without having to touch your files.

At all, I am open about all constructive comments to improve my work.

Regards
